app/views/hyrax/my/collections/_modal_collection_types_to_create.html.erb

```diff
@@ -12,7 +12,7 @@
           <% @collection_type_list_presenter.each do |row_presenter| %>
             <div class="radio radio-button-list">
               <label>
-                <input type="radio" name="collection_type"
+                <input type="radio" name="collection_type" value="<%= row_presenter.title.gsub(/\s+/, '') %>"
```

Comment:
Adding the title as the value is to support feature tests which need a way to select a radio button.